src/Host/Client/Impl/Encodings/REncodingsMap.cs

```diff
@@ -0,0 +1,1082 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.R.Host.Client.Encodings {
+    public static class REncodingsMap {
```

Comment:
Nope, as I mentioned in comments. May come useful at some point so - ifdef it out? Exclude?